libcxx/utils/ci/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.7'
+services:
+  libcxx-builder:
+    image: ghcr.io/libcxx/libcxx-builder
```

Comment:
Did you create a libcxx organization for this?  Is there any reason not to use the llvm org?

Reply:
I've had the org for years now after noticing it was unclaimed. I recently started using it to test and prototype self hosted github actions. It's a lot easier to iterate the much smaller org.

Github permissions still seem rather course grained, which might be an issue.

Reply:
OK, I really like the idea of using the github container registry for this, but I'm going to recommend you use the llvm org to host the packages for a few reasons:

- In the past the LLVM Foundation has discouraged sub-projects from creating their own spaces on GitHub separate from the main project.  Having a libcxx org for testing and prototype doesn't seem like a problem to me, but I think hosting official project images there may cross the line.
- Bandwidth charges: GitHub has recently extended it's free period for container storage and bandwidth, so this isn't an issue today, but in the future if they start charging, you will likely need to request a budget from the LLVM Foundation for this which would require moving the containers to the llvm org.  Also, you aren't charged for bandwidth when downloads come from GitHub Actions, but this only applies to containers in your current org from what I can tell.

I understand that it's easier to manage permissions using the libcxx org, but it sounds like there be specific fine-grained permissions just for the container registry, so I can look into setting that up for you.

Reply:
I don't disagree with anything you have said.

The libc++ org is better than the status quo of ldionne's personal dockerhub, but LLVM's org is the most appropriate.

Reply:
@tstellar One thing we're also trying to avoid is being too dependent on heavy processes or bottlenecks for making modifications of this kind. Right now with how permissions are set up in the LLVM org, we're not empowered to make these kinds of changes IIUC. With our own organization, we are.

I am not disagreeing that LLVM is the right place to do all of it (it totally is), but in the balance is also our ability to actually get stuff working, maintain them and solve our problems without having to wait for someone with enough admin privileges to set things up for us (even though you all have been amazingly responsive and helpful whenever we needed help). My request for admin-ships in the LLVM org was related to this amongst other things.